tests/test_routes.py

```diff
@@ -0,0 +1,60 @@
+from app.models.planet import Planet
+
+def test_get_all_planets_with_no_records(client):
+    # Act
+    response = client.get("/planets")
+    response_body = response.get_json()
+
+   # Assert
+    assert response.status_code == 200
+    assert response_body == []
+
+def test_get_planets(client, two_saved_planets):
```

Comment:
this test name could be more descriptive. `test_get_planet_by_id` since you are looking for planet 1